sky/cli.py

```diff
@@ -1406,9 +1409,10 @@ def _get_in_progress_spot_jobs(
     """
     num_in_progress_jobs = None
     try:
-        with ux_utils.suppress_output():
-            # Make the call slient
-            spot_jobs = core.spot_queue(refresh=False, skip_finished=True)
+        with sky_logging.silent():
```

Comment:
What's the diff b/t this and before?

Reply:
Are you saying `core.spot_queue()` now uses `rich`, so previous `suppress_output()` is non-ideal? Wdyt about making `core.*` not use `rich`? Programmatic APIs do not need and usually do not have such UX niceties, I think.

Reply:
Oh, I mean it will cause the rich status in the `cli.spot_queue` ([here](https://github.com/skypilot-org/skypilot/blob/24c1190644d36d0d9545e07988d7ebc4dc1886ca/sky/cli.py#L3312)) stop working, i.e. the spinner will stop after we enter the `suppress_output()` context.

Reply:
Oh I see. https://docs.python.org/3/library/contextlib.html#contextlib.redirect_stdout used as a ctx mgr still has global effects, which affect outer-scope logging. ` suppress_output()` seems removable now.